src/components/atoms/Text/style.ts

```diff
@@ -0,0 +1,82 @@
+import { Typography } from '@mui/material';
+import { styled } from '@mui/system';
+
+type TextStyleProps = {
+  $small?: boolean;
+  $color?: string;
+};
+
+export const H1 = styled(Typography)(({ $color }: TextStyleProps) => ({
+  fontFamily: 'Heebo, sansSerif',
+  fontStyle: 'normal',
+  fontWeight: '700',
+  fontSize: '34px',
+  lineHeight: '42px',
+  color: $color || '#000',
+}));
+
+export const H2 = styled(Typography)(({ $color }: TextStyleProps) => ({
+  fontFamily: 'Heebo, sansSerif',
+  fontStyle: 'normal',
+  fontWeight: '700',
+  fontSize: '28px',
+  lineHeight: '38px',
+  color: $color || '#000',
+}));
+
+export const H3 = styled(Typography)(({ $color }: TextStyleProps) => ({
+  fontFamily: 'Heebo, sansSerif',
+  fontStyle: 'normal',
+  fontWeight: '700',
+  fontSize: '25px',
```

Comment:
maybe, it would be better to replace abs units with relative units (rem for example)?